src/App.jsx

```diff
@@ -1,17 +1,25 @@
-import React, { useState } from 'react';
-import Data from './Data.json';
-
+import React, { useState, useCallback, useEffect } from 'react';
+import { v4 } from 'uuid';
 import Header from './Header';
 import ToDoList from './ToDoList';
 import ToDoForm from './ToDoForm';
 
 function App() {
-    const [toDoList, setToDoList] = useState(Data);
-    const [listId, setListId] = useState(toDoList.length);
+    return (
+        <div>
+            <div className="App">
+                <Header />
+                <Todos />
+            </div>
+        </div>
+    );
+}
+function Todos() {
+    const [toDoList, setToDoList] = useState([]);
 
-    const handleButton = (id) => {
+    const onRemoveTodo = (id) => {
```

Comment:
to be persistent with the `onAddTodo` function parameters (which now accepts task), let's also expect a tasks here (then you can grab its id). so `({ task })` or (task) as parameters